main.tex

```diff
@@ -18,6 +18,21 @@
 \begin{document}
 
 \maketitle
+\subsubsection Internetverbreitung
+
+In Indien gibt es 2015 schon 350 millionen Menschen\autocite{slideshareIndia} , welche das Internet aktiv nutzen, und davon sind 159 millionen per Handy im Internet unterwegs. Somit sind sie schon auf Platz 2 der aktiven Internetnutzer weltweit. So groß sich die Zahl auch anhören mag, das sind lediglich 27\% der Bevölkerung\autocite{InternetCountry}. 
+
+Laut einer Studie sei 2015 ein PC in Indien ein Luxusgut, welches durch die schnelle Entwicklung des Handys für die Jugend schnell die Bedeutung verlor, und Handys mit Internet in der Gesellschaft allgegenwärtig wurde. Durch die explosionsartige Verbreitung der neuen Möglichkeit mit anderen zu kommunizieren und sich auszutauschen bauten sich in kurzer Zeit große Gruppen im Internet auf, um sich über neue Technologien, Wissenswertes und Interessen auszutauschen. Dieser rapide Aufbau von Gesellschaften ist ein neues Phänomen, welches vorallem in "Entwicklungsländern" auftritt, in denen neue und vergleichsweise preiswerte Technologien den Zugang zum Internet ermöglichen.   \textcite{empowermentThroughFacebook}
+
+\medskip
+Die durchschnittliche Bandbreite des mobilen Netzes war mit 2.8 Mb/s der durchschnittlichen festen Internetleitung überlegen, denn diese betrug lediglich 2.3 Mb/s\autocite{slideshareIndia}.
+Da eine Prepaid Karte mit 1GB Datenvolum ca. 3.55\% des durchschnittlichen Monatsgehalt kosteten \autocite{broadbandAgency}, können sich hauptsächlich nur die Inder Internet leisten, welche in Stadtregionen leben, da dort das Gehalt weitaus höher ist.
+
+ Der Großteil der indischen Bevölkerung lebt aber im ländlichen Gebiet  und gerademal 31,2 \% \autocite{IndiaBevölkerung} in städtischen Gebieten. 
+Von den 350 millionen aktiven Internetnutzern leben also lediglich 61 millionen nicht in einer Stadt. Somit benutzten 2015 gerademal 7.5 \%, auf dem Land lebende Inder, aktiv das Internet \autocite{slideshareIndia}.
+Grund dafür ist vorallem die fehlende Abdeckung mobilen Internets in ländlichen Gebieten und der hohe Preis, was dazu führt, dass vorallem Jugendliche vom Land in städtische Gebiete ziehen, um dort bessere Bildung und bessere Arbeitsplätze zu beziehen und somit die ländlichen Gebiete noch mehr dem Armut verfallen. 
```

Comment:
`, was dazu führt, dass vorallem Jugendliche` ließt sich als ob die Jungendlichen wegen des teuren Internets in die Städte ziehen :)